tests/typescript_import_export/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,45 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`type-modifier.ts 1`] = `
+====================================options=====================================
+parsers: ["typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+export type { SomeThing };
+export type { A as B };
+export type { B as C } from './a';
+export type { foo } from 'bar';
+export type * from 'bar';
+export type { foo };
+
+// this should be treated as a normal import statement
+import type from './foo';
+
+import type { SomeThing } from "./some-module.js";
+import type { foo, bar } from 'baz';
+import type { foo as bar } from 'baz';
+import type * as foo from './bar';
+import type foo from 'bar';
+import type foo, { bar } from 'bar';
+
+=====================================output=====================================
+export type { SomeThing };
+export type { A as B };
+export type { B as C } from "./a";
+export type { foo } from "bar";
+export type * from "bar";
```

Comment:
@thorn0 Is this test needed? TS Playground says "`Only named exports may use 'export type'.`". 
https://www.typescriptlang.org/play/?ssl=1&ssc=1&pln=2&pc=1#code/KYDwDg9gTgLgBDAnmYcBUcBmUIFs4BEARgIZQEDcAUEA

Reply:
Let's remove this test. Babel 7.9.0 can't parse it.